Update/chapter 3

hyckomatej opened the pull request:
updated chapter 3

lukacan commented:
- Interacting with the Network
  - I would like to see how can I use solana cli to fetch for example account , to see the latest block , to maybe create a new keypair, to request aidrop etc etc.
- Transaction Lifecycle
  - I would like to see more clarification for the text written in the important. Why is there the word "sequentially ", arent the Solana transactions processed in parallel if possible ??
- I think that this statement is not true "An account cannot be created unless its initial balance is sufficient to be rent-exempt."
  - it is correct that accounts have to be rent exempt but you can create temporary accounts to use between Instructions in single Transaction.
- Update runtime policy based on this please https://docs.solanalabs.com/validator/runtime

lukacan reviewed (changes requested):
other than the previous comment , some small typos , but first version lgtm

hyckomatej commented:
- Interacting with the Network
Added Solana CLI commands but kept RPC requests. I think it would be a waste to delete them but ig we can if u wont like it.
- Transaction Lifecycle
Transactions are processed in parallel but this admonition refers to instructions being executed sequentially within a single transaction.
- Removed incorrect statement regarding rent exemption
- Update runtime policy based on this please https://docs.solanalabs.com/validator/runtime
I dont know whether u only wanted me to update the policy or add something else from the docs. Please take a look and let me know if it is ok.
